apps/web/components/v2/eventtype/EventAdvancedTab.tsx

```diff
@@ -313,45 +312,57 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupInfered
       />
       <hr />
       <Controller
-        name="seatsPerTimeSlot"
+        name="seatsPerTimeSlotEnabled"
```

Comment:
Added a pseudo-field that won't be send to the backend but will allow us to keep track of this state locally.